README.md

```diff
@@ -235,6 +235,17 @@ $ remme node-account get \
 }
 ```
 
+Transfer tokens from frozen to unfrozen reputation's balances (executable only on the machine which runs the node) — `remme node-account transfer-tokens-from-frozen-to-unfrozen`.
```

Comment:
Add please, `Transfer available tokens`, it would be more understandable to the user and change it's everywhere.

Reply:
@Alladin9393, done.